domains/authorization/Proxy.IntegrationTests/Swagger/SwaggerTestAuthHandler.cs

```diff
@@ -0,0 +1,31 @@
+using System.Security.Claims;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Proxy.IntegrationTests.Swagger;
+
+public class SwaggerTestAuthHandler : AuthenticationHandler<AuthenticationSchemeOptions>
+{
+    public const string AuthScheme = "Test";
+
+    [Obsolete("Obsolete")]
```

Comment:
Why obsolete?

Reply:
I don't know :D You tell me. You wrote the code :D